Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -0,0 +1,128 @@
+import UIKit
+import PayPalNativeCheckout
+import PayPalCheckout
+import PaymentsCore
+import Card
```

Comment:
Why does the PayPalViewModel need to import Card?

Reply:
@scannillo it was from old code. Wonder if there is a lint rule for unecessary imports?